### PR TITLE
Add folks to teams page

### PR DESCRIPTION
**Pull request by slorello89**
## Description

Adding Dwane, Chris, Michael, and I to the team page

## Deploy Notes

Notes regarding deployment the contained body of work. These should note any db migrations, etc.

### CODE REVIEW

File: config/team.yml [hunk 1]
```diff
@@ -89,4 +89,20 @@ current:
     email: 'bethany.loft@vonage.com'
     twitter: 'bb_loft'
     role: 'Junior Community Event Manager'
+  - name: 'Steve Lorello'
+    email: 'steve.lorello@vonage.com'
+    twitter: 'slorello'
+    role: '.NET Developer Advocate'
+  - name: 'Chris Tankersley'
+    email: 'chris.tankersley@vonage.com'
```
Comment (tbedford) [inline config/team.yml]: It looks like Chris and Dwane don't have avatars co figured. We use Gravatar. In Gravatar you need to specify the avatar based on email address if memory serves (been a while).

Comment (slorello89) [reply config/team.yml]: @dragonmantank  - @conshus can you update this comment here when you've configured yoru gravatars?

Comment (conshus) [reply config/team.yml]: Had to upload a picture from another computer to get it working. Let me know if it doesn't work.
image

Comment (dragonmantank) [reply config/team.yml]: Capture

I have one set up? Maybe it's a caching issue?

Comment (slorello89) [reply config/team.yml]: @tbedford Looks like they should be set now - can you check again?

thanks,

**Pull request merged**